### DISCUSSION TITLE
Read List Posters

### DISCUSSION THREAD

**Discussion by gardn701** [DieselTech/CBL-ReadingLists#44] [General]
Does anyone have any opinions on poster cards for the read lists?   Would it be nice to have them?  What would you like them to look like? 

I can make some, and I'm open to ideas about design.  Komga shows the first 4 issue covers in a grid for read lists, so when you have a lot it's not exactly informative or pretty to look at.  Posters should probably be fairly simple, just a way to visually tell what the read list is at a glance without reading the name.  Posters for recent modern events are probably the simplest as the publishers actually made some for advertising.

**Comment by flips22** [comment]
Yeah I think this would be great.  Komga has the ability to change the thumbnail right in the UI. I'm not sure about kavita.  

I just checked one of the generated thumbnails and it is 212x300, 96 dpi and 24 bit jpeg.  Probably safe to use that as our template size. I think naming it the same as the CBL file and we can put them in the library side by side with the CBL.

**Comment by rmleonard** [comment]
if needed - you could binhex the graphic and include it inside the XML
file....

On Wed, May 10, 2023 at 11:54 AM flips22 ***@***.***> wrote:

> Yeah I think this would be great. Komga has the ability to change the
> thumbnail right in the UI. I'm not sure about kavita.
>
> I just checked one of the generated thumbnails and it is 212x300, 96 dpi
> and 24 bit jpeg. Probably safe to use that as our template size. I think
> naming it the same as the CBL file and we can put them in the library side
> by side with the CBL.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/DieselTech/CBL-ReadingLists/discussions/44#discussioncomment-5864648>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACDSO7PCMIVGFBIWBCGKVITXFPP5VANCNFSM6AAAAAAX4UEXQU>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: <DieselTech/CBL-ReadingLists/repo-discussions/44/comments/5864648@
> github.com>
>


-- 
--------------
Sent from GMAIL web browser
***@***.***

**Reply by gardn701** [reply]
I'm in favor of any method that could have the image be auto imported.

**Comment by gardn701** [comment]
So just seeing if y'all have any opinions on what you'd prefer the aesthetics of these covers to look like.  For the master lists and character lists I was thinking of something simple with an evolution of the logos over time, and breaking up pieces into Ages > Year groups.  It's hard to know what looks best for people who use either light/dark themes too.  I'm still deciding how to spruce them up style-wise.

<img width="1148" alt="2023-05-16 (2)" src="https://github.com/DieselTech/CBL-ReadingLists/assets/79603982/920a336d-00b0-4cf6-b697-843c2388684f">

For the events, do you prefer seeing some sort of event advertisement with unique art or would you prefer something more textual and simple but different?   I came across the "The Fourth World By Jack Kirby" cover and I rather like its style.  I could go with that theme and create color variations for each event, but it would create a nice cohesion for all event read lists.  Which would you prefer?


![JackKirbysFourthWorld](https://github.com/DieselTech/CBL-ReadingLists/assets/79603982/fafde123-bce1-45be-9fe0-1147451d1af7)

**Reply by flips22** [reply]
Wow, those look great!  I like the uniformity that you have for the different ages.  For events, I think that would be tough. Each event's style is going to be very different from event to event, so to get them to look cohesive, you would largely rely on the text of the event with a somewhat generic background (I think). Having the thumbnail represent one of covers like you have I think would be best, mainly as I don't know how you would pull off a cohesive collection of event thumbnails.

I like the idea of embedding into the XML. That's what I do when I make a html of a CBL file. The downside is that we would need the developers to adopt that scheme. With a loose JPEG, it can be implemented right now.
